### ISSUE TITLE
"hugo mod" command requires go

### ISSUE THREAD

**Issue by ebousse** [trenton42/hugo-extended-docker#1]
I'm trying to run the following valid hugo command:
```sh
docker run --rm -v $PWD:/mydata:z -w /mydata docker.io/trenton/hugo-extended:0.80.0 hugo mod init my-site
```
and I get as an answer:
```
Error: failed to download modules: exec: "go": executable file not found in $PATH
```
I believe this means the image should include `go`?

and thanks for this nice image!

**Comment by trenton42** [comment]
Yes, I will check this out! Sorry for missing this issue.

**Comment by trenton42** [comment]
@ebousse This should be fixed in v0.83.0
